### PR TITLE
change border color

### PR DESCRIPTION
**Pull request by viabadia**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <style>
         body{
             background-color: orange;
-            border: 10px solid skyblue;
```
Comment (maryammdot) [inline index.html]: hey, your line is cool

File: index.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <style>
```
Comment (maryammdot) [inline index.html]: Hey, move styles to file

Comment (viabadia) [reply index.html]: This is true, but both have done that and alegria Macarena, how we proced then?

Comment (maryammdot) [reply index.html]: dont' worry, be happy, I'll do it.

**Pull request merged**